test/Dialect/OM/round-trip.mlir

```diff
@@ -252,9 +252,15 @@ hw.module @PathModule() {
   %wire = hw.wire %wire sym @wire : i1
 }
 // CHECK-LABEL: @Path
-om.class @Path() {
-  // CHECK: %0 = om.path reference @HierPath
+om.class @Path(%p: !om.path, %p2: !om.path) {
+  // CHECK: %[[v0:.+]] = om.path reference @HierPath
   %0 = om.path reference @HierPath
+  // CHECK:  om.path_append %p, %[[v0]]
+  %1 = om.path_append %p, %0
+  %2 = om.path_append %p, %p2
+  om.class.field @path1, %2 : !om.path
+  // CHECK:  %[[v2:.+]] = om.path_append %p, %p2
+  // CHECK:  om.class.field @path1, %[[v2]] : !om.path
```

Comment:
some of the CHECK are leading, other are trailing. Seems like the above tests are in the leading style. Also seems like we don't need everything here, so I suggest the following:
```suggestion
om.class @Path(%p: !om.path) {
  // CHECK: %[[v0:.+]] = om.path reference @HierPath
  %0 = om.path reference @HierPath
  // CHECK:  %[[v1:.+]] = om.path_append %p, %[[v0]]
  %1 = om.path_append %p, %0
  // CHECK:  om.class.field @path1, %[[v1]] : !om.path
  om.class.field @path1, %1 : !om.path
```